src/blocks/gallery-carousel/index.js

```diff
@@ -52,17 +52,6 @@ const settings = {
 		html: false,
 		coBlocksSpacing: true,
 	},
-	example: {
-		attributes: {
-			gridSize: 'lrg',
-			gutter: 5,
-			images: [
-				{ url: 'https://s.w.org/images/core/5.3/Biologia_Centrali-Americana_-_Cantorchilus_semibadius_1902.jpg' },
-				{ url: 'https://s.w.org/images/core/5.3/Glacial_lakes,_Bhutan.jpg' },
-				{ url: 'https://s.w.org/images/core/5.3/MtBlanc1.jpg' },
-			],
-		},
-	},
```

Comment:
So I was thinking about this and how we are now going and adding examples back where we did not have them. Here we are removing the example and the reason is made clear within this PR but I think the reason initially looking at the code is not clear. For that reason, I would suggest adding an additional comment here specifying the reason that no example is present. Something like:


```
	attributes,
	variations,
	icon: <Icon icon={ icon } />,
	keywords: [...],
	// Diable reason: link to the issue in GB
        // example,
```